### ISSUE TITLE
[19.01.23]  Grad-CAM: Visual Explanations from Deep Networks via Gradient-based Localization

### ISSUE THREAD

**Issue by SSUHan** [SSUHan/PaparReviews#16]
# [Grad-CAM: Visual Explanations from Deep Networks via Gradient-based Localization](https://arxiv.org/abs/1610.02391)
- Accepted Conference Name & Year : ex) arXiv:1610.02391
- 1st Author Name & Institute : Ramprasaath R. Selvaraju, Virginia Tech

![2019-01-23 6 23 06](https://user-images.githubusercontent.com/8327479/51596286-f4141880-1f3b-11e9-98cf-1bd97db3ab04.png)

## Keywords
- Visualize of CNN without change network architecture.
- Assessing Model Trust
- Weakly supervised localization
- Gradient based Activation Map

## Contribution 
- Producing ‘visual explanations’ for decisions from a large class of Convolutional Neural Network (CNN)-based models
- Help identify dataset bias (Section 6.3) and lend insight into failures of current CNNs (Section 6.1), showing that seemingly unreasonable predictions have reasonable explanations.
- Proposing Grad-CAM, a class-discriminative localization technique that can generate visual explanations from any CNN-based network without requiring architectural changes or re-training.

## Proposed Architecture
![2019-01-23 6 18 54](https://user-images.githubusercontent.com/8327479/51596030-646e6a00-1f3b-11e9-91ec-8e7ca5a7ed6d.png)
- CNN 의 마지막 레이어와 Softmax 에서 해당 클래스를 나타내는 노드에 영향을 미치는 정도(weights) 를 이용해서, 어느 위치의 feature 가 어떤 클래스를 결정짓는데 지대한 영향을 끼치는지를 확인할 수 있다. 
- 학습을 위해서는 네트워크 전체가 미분가능한 구조이므로, 이렇게하면, 기존 네트워크의 구조를 바꾸지 않고도, 모델이 무엇을 보고 결과를 나타내는지를 직관적으로 확인할 수 있다. (천재?)

![2019-01-23 6 19 35](https://user-images.githubusercontent.com/8327479/51596069-794afd80-1f3b-11e9-9641-a0594bf7a15c.png)
- 작성하기도 그렇게 어렵지는 않음

## Dataset
- COCO, 
- imageNet
- PASCAL
## Valuable Relative Works
- 
- http://hugrypiggykim.com/2018/03/28/grad-cam-gradient-weighted-class-activation-mapping/
